install.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 REPO_OWNER=gosh-sh
 REPO=gosh
-RELEASE=rc-3.0.17
+TAG=latest
```

Comment:
This needs to be configured in the release pipeline

Reply:
I'm not quite sure that pipeline can push changes to the repo, or it will require keys